### PR TITLE
fix scan modality

### PR DESCRIPTION
**Pull request by cremebrule**
Fix scan modality to (a) not crash, and (b) be flexibly compatible with multiple channels

### CODE REVIEW

File: robomimic/models/base_nets.py
Comment (amandlek) [inline robomimic/models/base_nets.py:856]: should we assert these keys do not already exist in `conv_kwargs` or are they meant to be overwritten?

**Review by amandlek — COMMENTED** [review]
LGTM in general! I'm wondering how we missed this before - I thought the MOMART experiments used the "scan" modality with the 1D conv and we tried reproducing results before merging that PR. I left one minor comment as well, and can approve the PR after that's resolved.

File: robomimic/models/base_nets.py
Comment (cremebrule) [reply robomimic/models/base_nets.py:856]: Great question! If you see in the lines above, they can't actually exist already in `conv_kwargs` precisely because those kwargs are explicitly listed in the constructor (that's the reason for the error!)

**Comment by cremebrule** [comment]
> LGTM in general! I'm wondering how we missed this before - I thought the MOMART experiments used the "scan" modality with the 1D conv and we tried reproducing results before merging that PR. I left one minor comment as well, and can approve the PR after that's resolved.

Yeah, I'm not sure how either. I know for MOMART I hardcoded the number of input channels to be 1, but there's no reason why that should be the case (e.g.: a 16-channel LIDAR vs. 1-channel).

In terms of things breaking, yeah I'm not entirely sure what changed -- my initial guess is that perhaps the torch implementation changed slightly, resulting in a different interface? very strange though

**Review by amandlek — APPROVED** [review]

**Pull request merged**
